ethcore/src/executive.rs

```diff
@@ -34,7 +34,7 @@ pub use types::executed::{Executed, ExecutionResult};
 /// Maybe something like here: `https://github.com/ethereum/libethereum/blob/4db169b8504f2b87f7d5a481819cfb959fc65f6c/libethereum/ExtVM.cpp`
 const STACK_SIZE_PER_DEPTH: usize = 24*1024;
 
-const WASM_MAGIC_NUMBER: [u8; 4] = b"\0asm";
+const WASM_MAGIC_NUMBER: &'static [u8; 4] = b"\0asm";
```

Comment:
Alternatively (it doesn't matter in this case, but it might in the future):

```rust
const WASM_MAGIC_NUMBER: [u8; 4] = *b"\0asm";
```

Also, Rust will automatically infer `'static` in `const` and `static` items (so you can write `&[u8; 4]` for the type)